finetune/config.py

```diff
@@ -11,7 +11,7 @@
 
 import finetune
 from finetune.errors import FinetuneError
-from finetune.base_models import GPTModel, GPT2Model, BERT, TextCNN
```

Comment:
I don't see a reason why we'd need this??

Reply:
Yeah this is unnecessary.